src/components/inputs/DatePickerComponent.tsx

```diff
@@ -8,14 +9,14 @@ import { SecondaryBtn } from '../buttons/SecondaryBtn'
 import { useState } from 'react'
 
 interface Prop {
-  getDate: (value: string) => void
+  getDate: (value: unknown) => void
```

Comment:
Looking at its usage, this seems very weakly typed.

Reply:
fixed in the commit below